README.md

```diff
@@ -9,7 +9,7 @@ Java, version 11 or newer, and Apache Maven are required for building this proje
 Build the project like this:
 
 ```
-$ mvn clean verify
+$ mvn clean install
```

Comment:
When I ran `mvn clean verify` it wouldn't work as it told me I didn't have a specific (kroxylicious) related component installed. I'm afraid I forgot to note exactly what one it was, though - but switching this to install worked for me.

Reply:
hmm `verify` should be enough, it loosely implies we've not got the module relationships right in maven. Having said that I've no problems with having `install` in the readme.

Reply:
Yeah, I was wondering about this too. Part of this aggregator build is a plug-in which then is applied to the actual proxy module. When not installing the former, a build of only the latter will fail. So all in all, `install` should be the more robust choice and it makes sense to have it here.

Reply:
I've raised https://github.com/kroxylicious/kroxylicious/issues/106 to cover fixing `verify`.